android-components/components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/storage/SyncedTabsStorage.kt

```diff
@@ -42,6 +43,19 @@ class SyncedTabsStorage(
 ) : SyncedTabsProvider {
     private var scope: CoroutineScope? = null
 
+    // TODO; Pass this value in from the constructor so that it can be set from the Fenix integration side.
+    private val maxActiveTime: Long = TimeUnit.DAYS.toMillis(14)
+
+    /**
+     * TODO; This needs to be upstreamed from [org.mozilla.fenix.tabstray.ext.TabSessionState]. We
+     * can't just copy it over.
+     */
+    fun TabSessionState.isActive(maxActiveTime: Long): Boolean {
+        val lastActiveTime = maxOf(lastAccess, createdAt)
+        val now = System.currentTimeMillis()
+        return (now - lastActiveTime <= maxActiveTime)
+    }
```

Comment:
I think we could just go ahead and move this to a top-level extension function in the [TabSessionState](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TabSessionState.kt) file. That way, it could be shared between here and the tabs tray more easily.